Mlem/App/Views/Pages/PostEditor/PostEditorView.swift

```diff
@@ -187,24 +208,34 @@ struct PostEditorView: View {
     
     @ViewBuilder
     var targetSelectionView: some View {
-        VStack(spacing: Constants.main.standardSpacing) {
-            ForEach(targets, id: \.id) { target in
-                HStack(spacing: 0) {
-                    PostEditorTargetView(target: target)
-                        .frame(maxWidth: .infinity, alignment: .leading)
-                    if targets.count > 1 {
-                        Button("Remove", systemImage: Icons.closeCircleFill) {
-                            if let index = targets.firstIndex(where: { $0.id == target.id }) {
-                                targets.remove(at: index)
+        VStack(alignment: .leading, spacing: Constants.main.standardSpacing) {
+            if let postToEdit {
+                FullyQualifiedLinkView(
+                    entity: postToEdit.community,
+                    labelStyle: .medium,
+                    showAvatar: true
+                )
+                .padding(.horizontal)
+                Divider()
+            } else {
+                ForEach(targets, id: \.id) { target in
+                    HStack(spacing: 0) {
+                        PostEditorTargetView(target: target)
+                            .frame(maxWidth: .infinity, alignment: .leading)
+                        if targets.count > 1 {
+                            Button("Remove", systemImage: Icons.closeCircleFill) {
+                                if let index = targets.firstIndex(where: { $0.id == target.id }) {
+                                    targets.remove(at: index)
+                                }
```

Comment:
This might be cleaner if the original `ForEach` uses `ForEach(Array(targets).enumerated) { index, target in`

Reply:
Nice, that's much cleaner! 👍